wiki/History_of_osu!/2010/en.md

```diff
@@ -1 +1,68 @@
 # History of osu! 2010
```

Comment:
while that's a start (and a good one), I'm sure there's still a lot to go through

```suggestion
---
stub: true
---

# History of osu! 2010
```

Reply:
Out of curiousity, do you have anything specific in mind that is not included here?

Reply:
not really, at least not at the moment, just that the article feels too slim -- surely there has been a lot going on in 2010